android-sdk/src/test/java/org/sagebionetworks/bridge/sdk/android/ExampleUnitTest.java

```diff
@@ -10,6 +19,9 @@
 public class ExampleUnitTest {
   @Test
   public void addition_isCorrect() throws Exception {
+
+    Logger log = LoggerFactory.getLogger(ExampleUnitTest.class);
```

Comment:
I couldn't figure out how to configure logback to log to console, but by also adding simple-logger (which you're not supposed to do, since you're supposed to provide one unique impl) it works for me.